web/packages/shared/components/UnifiedResources/unifiedStyles.css

```diff
@@ -20,16 +20,9 @@
     TODO (avatus): DELETE if we ever upgrade to v6
 */
 .ContainerContext {
-  min-width: 800px;
   container-type: inline-size;
 }
 
-.ResourcesContainer {
-  @container (min-width: 1600px) {
-    grid-template-columns: repeat(4, minmax(400px, 1fr));
-  }
-}
```

Comment:
IMO this is unnecessary, we set `max-width: 1800px` on the container, so we won't have more than 4 columns anyway.